plugins/outputs/whatap/whatap.go

```diff
@@ -0,0 +1,382 @@
+package whatap
+
+import (
+	"fmt"
+	"log"
```

Comment:
Please do not use `log` but rather use the `telegraf.Logger` facility. If you add 
```golang
Log telegraf.Logger `toml:"-"`
```
to your struct, it will automatically injected.

Reply:
Edited and applied. We ask for confirmation.